migrations/sql/R__stored_procedure_for_bond_etl.sql

```diff
@@ -0,0 +1,356 @@
+		-- "temp" varchar NULL,--?? Breakdown of permit number
+);
+-- columns with no source
+-- bond.closed_note
+
+
+	------------------- UPSERT RECORDS
+	SELECT count(*) FROM ETL_BOND into tmp1;
+
+	with upserted_etl_bond as (
+	INSERT INTO ETL_BOND (
+		sec_cid,
+		permit_no ,
+		mms_address,
+		cmp_nm,
+		note1,
+		sec_amt ,
+		core_bond_type_code,
+		sec_typ,
+		descript ,
+		"comment",
+		invloc ,
+		iaddr1 ,
+		iaddr2 ,
+		iaddr3 ,
+		ipost_cd ,
+		project_no ,
+		"status",
+		return_dt,
+		cnt_dt,
+		etl_create_date,
+		etl_update_date)
+	SELECT
+		sec_cid,
+		replace(REPLACE(permit_no,' ',''),'--','-'),
+		CONCAT_WS(' ', TRIM(addr1),TRIM(addr2),TRIM(addr3), TRIM(post_cd)),
+		RTRIM(coalesce(nullif(TRIM(cmp_nm),''),TRIM(last_nm)),','),-- 1 record ends with a comma
+		TRIM(note1),
+		sec_amt,
+		case
+			when TRIM(sec_typ) = 'Asset Security Agreement' then 'ASA'
+			when TRIM(sec_typ) = 'Cash' then 'CAS'
+			when TRIM(sec_typ) = 'Confiscation' then 'CAS'
+			when TRIM(sec_typ) = 'Letter of Credit' then 'ILC'
+			when TRIM(sec_typ) = 'Qualified Env. Trust' then 'QET'
+			when TRIM(sec_typ) = 'Safekeeping Agreement' then 'SAG'
+			when TRIM(sec_typ) = 'Receipt and Agreement' then 'SAG'
+			when TRIM(sec_typ) = 'Surety Bond' then 'SBO'
+			when TRIM(sec_typ) = 'Recl. Fund' then 'STR'
+			when TRIM(sec_typ) = 'Performance Bond' then 'PFB'
+		end as core_bond_type_code,
+		TRIM(sec_typ),
+		TRIM(descript),
+		CONCAT(TRIM(comment1),' ', TRIM(comment2)),
+		TRIM(invloc),
+		TRIM(iaddr1),
+		TRIM(iaddr2),
+		TRIM(iaddr3),
+		TRIM(ipost_cd),
+		TRIM(project_no),
+		"status",
+		return_dt,
+		cnt_dt,
+		now(),
+		now()
+	from mms.secsec sec
+	where TRIM(sec.sec_cid) != ''
+	and replace(REPLACE(permit_no,' ',''),'--','-') in (select permit_no from permit)
+	and sec_typ not in ('ALC', '')
+	ON CONFLICT (sec_cid)
+	DO
+		UPDATE
+			SET
+				sec_amt=excluded.sec_amt,
+				core_payer_party_guid = case
+					when ETL_BOND.cmp_nm != EXCLUDED.cmp_nm then null
+					else ETL_BOND.core_payer_party_guid
+					end,
+				core_bond_type_code=excluded.core_bond_type_code,
+				descript= TRIM(excluded.descript),
+				etl_update_date=excluded.etl_update_date,
+				invloc=excluded.invloc,
+				iaddr1=excluded.iaddr1,
+				iaddr2=excluded.iaddr2,
+				iaddr3=excluded.iaddr3,
+				ipost_cd=excluded.ipost_cd,
+				note1=excluded.note1,
+				"status"=excluded.status,
+				cnt_dt=excluded.cnt_dt,
+				project_no=excluded.project_no,
+				return_dt=excluded.return_dt
+	returning *
+	)
+	SELECT count(*) FROM upserted_etl_bond into tmp3;
+
+	SELECT count(*) FROM ETL_BOND into tmp2;
+
+	RAISE NOTICE '....# of ETL_BOND record before.. %', tmp1;
+	RAISE NOTICE '....# of ETL_BOND records after.. %', tmp2;
+	RAISE NOTICE '....# of ETL_BOND records upserted. %', tmp3;
+
+	--try using classifcation function
+	update ETL_BOND set
+	core_party_type = permitee_party_type(cmp_nm)--permittee_party_type defined in R__stored_procedure_for_permit_but_better.sql
```

Comment:
There are still some instances of "permittee" spelled incorrectly. Do you mind fixing all of those?